File: templates/go/template.go
// Minimal Go competitive-programming template.
package main

import (
	"bufio"
	"os"
)

func solve(in *bufio.Reader, out *bufio.Writer) {
	// Parse the problem-specific input and write the answer.
}

func main() {
	in := bufio.NewReader(os.Stdin)
	out := bufio.NewWriter(os.Stdout)
	defer out.Flush()
	solve(in, out)
}

File: templates/rust/template.rs
// Minimal Rust 2021 competitive-programming template.

use std::io::{self, Read};

fn solve(input: &str) -> String {
    let _ = input;
    String::new()
}

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    print!("{}", solve(&input));
}
